spec/controllers/idv/sessions_controller_spec.rb

```diff
@@ -9,29 +9,43 @@
   end
 
   describe '#destroy' do
-    let(:idv_session) { double }
-    let(:flow_session) { {} }
-    let(:pii) { { first_name: 'Jane' } }
-
     before do
       allow(idv_session).to receive(:clear)
       allow(subject).to receive(:idv_session).and_return(idv_session)
       controller.user_session['idv/doc_auth'] = flow_session
       controller.user_session['idv/in_person'] = flow_session
+      controller.user_session['idv/inherited_proofing'] = flow_session
       controller.user_session[:decrypted_pii] = pii
     end
 
-    it 'deletes idv session' do
-      expect(idv_session).to receive(:clear)
+    let(:idv_session) { double }
+    let(:flow_session) { { x: {} } }
```

Comment:
Continuing from previous comments: I'm not expecting we need anything here. The following suggestion passes for me locally. Should it be failing? Otherwise, any problems with keeping it empty as it was before?

```suggestion
    let(:flow_session) { {} }
```

Reply:
Oh, actually, I think I see what you're getting at with this, and the tests before may have been insufficient.

Reply:
Yeah, this makes sense and is a good improvement over what we had previously, since previously we weren't reliably testing the controller, as the initial stubbed value would have been blank anyways, and technically the controller could be doing nothing and it would have still passed.